packages/beeq/src/components/slider/bq-slider.tsx

```diff
@@ -281,6 +328,20 @@ export class BqSlider {
     );
   };
 
+  private renderTooltip = (value: number, thumbPosition: number): HTMLBqTooltipElement => (
+    <bq-tooltip
+      class="absolute"
+      exportparts="base,trigger,panel"
+      alwaysVisible={this.isTooltipAlwaysVisible}
+      visible
+      distance={this.enableValueIndicator ? 6 : 16}
+      style={{ left: `${thumbPosition}px`, fontVariant: 'tabular-nums' }}
+    >
+      <div class={{ 'absolute h-1 w-1': true, '-z-[1]': this.tooltipAlwaysVisible }} slot="trigger" />
```

Comment:
You can use `size-*` to set same `width` and `height`.

```suggestion
      <div class={{ 'absolute size-1': true, '-z-[1]': this.tooltipAlwaysVisible }} slot="trigger" />
```

![CleanShot 2024-04-12 at 18 37 40@2x](https://github.com/Endava/BEEQ/assets/328492/e91d3232-339f-4389-b6ac-435ce5a2883a)